osidb/models/tests/test_affect.py

```diff
@@ -80,3 +80,18 @@ def test_resolved_dt_auto_update(
         affect.save()
         assert affect.is_resolved == is_resolved_2
         assert affect.resolved_dt or not is_resolved_2
+
+    @pytest.mark.parametrize(
+        "purl,ps_component",
+        [
+            (
+                "pkg:rpm/redhat/nginx@1.14.1-9.module+el8.0.0+4108+cba21616?arch=x86_64&rpmmod=mainstream",
+                "mainstream/nginx",
+            )
+        ],
+    )
```

Comment:
nitpick: parametrizing seems a bit odd when there is only one case to test

Reply:
Maybe we could test also the other case for the `oci` type and maybe one for normal case when no special handling is applied, then it would make sense to parametrize this test